src/App.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
-import {BounceLoader} from 'react-spinners'
+// import {BounceLoader} from 'react-spinners' // bounder loader is not being used. What is it's intention?
 import "./App.css";
 import { BrowserRouter } from "react-router-dom";
 import Routes from "./components/Routes";
 import Nav from "./components/Nav";
-import Footer from "./components/Footer";
+// import Footer from "./components/Footer";
 
 function App() {
   return (
     <div>
       <BrowserRouter>
         <Nav />
         <Routes />
-        <Footer />
+        {/* <Footer /> */}
```

Comment:
nit: I removed the footer, i felt like it serve no purpose. I will continue this trend you are more than welcome to bring it back.